app/components/Chatbot/ChatEntry.tsx

```diff
@@ -1,5 +1,5 @@
-import {ComponentType} from 'react'
-import {Link} from '@remix-run/react'
+import { ComponentType } from 'react'
```

Comment:
What was the purpose of adding all these spaces?

Reply:
probably auto import + the editor doing it. @varchanaiyer, make sure to run `npm run lint:fix` before pushing, as that will both make the code style consistent and notify you of any errors